compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1779,6 +1795,10 @@ class OMR_EXTENSIBLE CodeGenerator
    void incOutOfLineColdPathNestedDepth(){_outOfLineColdPathNestedDepth++;}
    void decOutOfLineColdPathNestedDepth(){_outOfLineColdPathNestedDepth--;}
 
+   bool getIsInWarmCodeCache() {return _flags2.testAny(IsInWarmCodeCache);}
+   void setIsInWarmCodeCache() {_flags2.set(IsInWarmCodeCache);}
+   void resetIsInWarmCodeCache() {_flags2.reset(IsInWarmCodeCache);}
```

Comment:
What is this flag used for?  I don't see a use of it in this PR.

If it is needed, what does it mean exactly?  That is, when used, someone would be asking the CodeGenerator object if "IsInWarmCodeCache" which doesn't make sense to me unless you supply a code address.

Reply:
Yeah, I should document. It's used by instruction selection to identify the stage it is in. Can be used for inserting corresponding debug counters for example. Currently, only used by openj9.

Reply:
Done

Reply:
Yes, agree. This name will be more descriptive.

Reply:
Renamed as above.